packages/forms/demo/fileUpload.stories.mdx

```diff
@@ -0,0 +1,53 @@
+import { Meta, ArgsTable, Canvas, Story } from '@storybook/addon-docs';
+import { FileUpload, Field, Label, Hint, Message } from '@zendeskgarden/react-forms';
+import { FileUploadStory } from './stories/FileUploadStory';
+
+<Meta
+  title="Packages/Forms/FileUpload"
+  component={FileUpload}
+  subcomponents={{ Field, Label, Hint, Message }}
+/>
+
+# API
+
+<ArgsTable />
+
+# Demo
+
+<Canvas>
+  <Story
+    name="FileUpload"
+    args={{
+      children: 'Drag files here or click to upload',
+      label: 'Label',
+      isLabelRegular: false,
+      isLabelHidden: false,
+      hasHint: true,
+      hint: 'Hint',
+      hasMessage: true,
+      message: 'Message'
+    }}
+    argTypes={{
+      label: { name: 'children', table: { category: 'Label' } },
+      isLabelRegular: { name: 'isRegular', table: { category: 'Label' } },
+      isLabelHidden: { name: 'hidden', table: { category: 'Label' } },
+      hint: { name: 'children', table: { category: 'Hint' } },
+      message: { name: 'children', table: { category: 'Message' } },
+      validation: {
+        control: { type: 'radio', options: ['success', 'warning', 'error'] },
+        table: { category: 'Message' }
+      },
+      hasHint: { name: 'Hint', table: { category: 'Story' } },
+      hasMessage: { name: 'Message', table: { category: 'Story' } }
+    }}
+    parameters={{
+      design: {
+        allowFullscreen: true,
+        type: 'figma',
+        url: 'https://www.figma.com/file/6g87L4FdKZTA3knt3Rsfdx/Garden?node-id=103%3A20268'
+      }
+    }}
+  >
+    {args => <FileUploadStory {...args} />}
```

Comment:
Anyone else having issues triggering the file upload UI? I'm clicking the dropzone area but nothing seems to happen.

Reply:
The `FileUpload` demo is meant to only show that component in isolation with no amendments or alterations. To see the component in action (combined with `react-dropzone`) go to the `Packages/Forms/[patterns]` section.

See https://github.com/zendeskgarden/react-components/blob/c31bae4face4e57cfe40c3f03d2c80827a0c09f4/docs/demo.md

> The development goal for demos is to render components in isolation with the least amount of structural scaffolding together with the greatest amount of control for content and flexibility.

Also take special note of the "Patterns" section at the bottom of that documentation.